src/app_mode.c

```diff
@@ -23,6 +23,7 @@ typedef struct {
 
 typedef struct {
     uint8_t secret;
+    uint8_t blind;
```

Comment:
would it be possible to use a bitfield here instead?

Reply:
Nice comment for the future, let's use it in future implementations 👌🏻